lib/game_team_manager.rb

```diff
@@ -15,7 +15,48 @@ def generate_game_teams(location) # I need a test
     end
     array
   end
+  
+  def goal_avg_per_team(team_id, home_away)
```

Comment:
I think you're spot on with this. It makes more sense to have these in GameTeamsManager. 👍